saleor/product/migrations/0108_auto_20191003_0422.py

```diff
@@ -21,12 +21,15 @@ def remove_duplicated_attribute_values(apps, schema_editor):
         values_to_be_removed = attribute_values[1:]
         with transaction.atomic():
             for value_to_be_removed in values_to_be_removed:
-                invalid_assigned_variants = (
+                invalid_assigned_attributes = list(
                     value_to_be_removed.assignedvariantattribute_set.all()
                 )
-                for invalid_assigned_variant in invalid_assigned_variants:
-                    invalid_assigned_variant.values.clear()
-                    invalid_assigned_variant.values.add(final_value)
+                invalid_assigned_attributes.extend(
+                    list(value_to_be_removed.assignedproductattribute_set.all())
+                )
+                for invalid_assigned_attribute in invalid_assigned_attributes:
+                    invalid_assigned_attribute.values.clear()
```

Comment:
What if the user assigned other values to this product attribute? (multi choice input type)